vignettes/introduction.Rmd

```diff
@@ -235,6 +235,29 @@ This is okay for testing things out and may even be necessary for smaller datase
 But in general it may be better to have larger numbers for these parameters; 
 we think the defaults are a good starting point [@topcuoglu_framework_2020].
 
+### Custom training indices
+
+When `training_frac` is a fraction between 0 and 1, a random sample of
+observations in the dataset are chosen for the training partition to satisfy the
+`training_frac`. However, in some cases you might wish to control exactly which
+observations are in the training partition. You can instead assign
+`training_frac` a vector of indices that correspond to which rows of the dataset
+should go in the training fraction of the train/test split.
```

Comment:
And everything else goes into the testing fraction?

Reply:
Yep!

Reply:
Cool, just wanted to make sure

Reply:
We could add that:
```suggestion
should go in the training fraction of the train/test split 
(all remaining sequences will go in the testing fraction).
```

Reply:
Perfect

Reply:
Adding this here: https://github.com/SchlossLab/mikropml/pull/282